### PR TITLE
Moved packaging of drivers for Cloud Data Fusion to a separate profile

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

**Comment by bdmogal** [comment]
@kurtisvg could you PTAL?

File: pom.xml [hunk 1]
```diff
@@ -237,6 +237,34 @@
...
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>2.4.1</version>
+            <configuration>
+              <!-- get all project dependencies -->
+              <descriptorRefs>
+                <descriptorRef>jar-with-dependencies</descriptorRef>
```
Comment (kurtisvg) [inline pom.xml]: nit: Should this be `jar-with-driver-and-dependencies`

Comment (bdmogal) [reply pom.xml]: yeah, I tried that, but didn't work. Apparently this is not free-form text. Here's the error I got:
{code}
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.4.1:single (make-assembly) on project jdbc-socket-factory-parent: Error reading assemblies: Descriptor with ID 'jar-with-driver-and-dependencies' not found -> [Help 1]
{code}

Comment (kurtisvg) [reply pom.xml]: Ah yup: https://maven.apache.org/plugins/maven-assembly-plugin/descriptor-refs.html#jar-with-dependencies

Ok, thanks

**Comment by bdmogal** [comment]
@kurtisvg thanks for merging this. Can we plan on doing a release to make this available to users?

**Comment by kurtisvg** [comment]
Releasing a new version doesn't really make sense as there aren't any changes since v1.15. I can just add some jar-with-drivers-and-dependencies to the v1.15 release - will that work?

**Comment by bdmogal** [comment]
Yes, as long as it is hosted somewhere, its good enough.

**Comment by kurtisvg** [comment]
Ok, added to the releases page: https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/releases/tag/v1.0.15

Please verify they work correctly for Cloud Data Fusion.

**Pull request merged**